collect_app/src/test/java/org/odk/collect/android/external/AndroidShortcutsActivityTest.kt

```diff
@@ -0,0 +1,80 @@
+package org.odk.collect.android.external
+
+import android.app.Application
+import android.net.Uri
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.test.espresso.matcher.RootMatchers.isDialog
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.whenever
+import org.odk.collect.android.formlists.blankformlist.BlankFormListItem
+import org.odk.collect.android.formlists.blankformlist.BlankFormListViewModel
+import org.odk.collect.android.formmanagement.FormsDataService
+import org.odk.collect.android.injection.config.AppDependencyModule
+import org.odk.collect.android.projects.ProjectsDataService
+import org.odk.collect.android.support.CollectHelpers
+import org.odk.collect.android.utilities.ChangeLockProvider
+import org.odk.collect.android.utilities.FormsRepositoryProvider
+import org.odk.collect.android.utilities.InstancesRepositoryProvider
+import org.odk.collect.androidtest.ActivityScenarioLauncherRule
+import org.odk.collect.async.Scheduler
+import org.odk.collect.formstest.InMemInstancesRepository
+import org.odk.collect.settings.SettingsProvider
+import org.odk.collect.testshared.Assertions
+
+@RunWith(AndroidJUnit4::class)
+class AndroidShortcutsActivityTest {
```

Comment:
Could we not test this in `AndroidShortcutsTest` instead by adding a match exactly project and then running background work once the shortcuts screen is open? That way we don't need a new test file or a lower level test that's in any way tied to the implementation.

Reply:
@grzesiek2010 just waiting for a response here!